### PR TITLE
Site install process fixes during bootstrap

### PR DESCRIPTION
**Pull request by AronNovak**
#658

### CODE REVIEW

File: .travis.template.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ env:
 
 before_install:
   - export PATH="$HOME/.config/composer/vendor/bin:$PATH"
+  - tmpdaemon=$(mktemp)
```
Comment (amitaibu) [inline .travis.template.yml:19]: What is it doing?

File: robo-components/DeploymentTrait.php [hunk 2]
```diff
@@ -562,7 +562,9 @@ public function deployPantheonInstallEnv(string $env = 'qa', string $pantheon_na
     }
 
     $task
+      ->exec("terminus connection:set $pantheon_terminus_environment sftp")
```
Comment (amitaibu) [inline robo-components/DeploymentTrait.php:565]: Worth a comment

**Review by amitaibu — APPROVED** [review]
Thanks

File: .travis.template.yml (hunk 1 above)
Comment (AronNovak) [reply .travis.template.yml:19]: It just uses a temporary file to do manipulations on `/etc/docker/daemon.json` . We can't redirect to the currently manipulated file, as the shell makes it empty as a very first step.

File: robo-components/DeploymentTrait.php [hunk 3]
```diff
@@ -562,7 +562,9 @@ public function deployPantheonInstallEnv(string $env = 'qa', string $pantheon_na
     }
 
     $task
```
Comment (AronNovak) [inline robo-components/DeploymentTrait.php:564]:
```suggestion
    // Drupal checks is the settings.php is writable. With the connection mode switch, we can fulfill this.
    $task
```

File: robo-components/DeploymentTrait.php (hunk 2 above)
Comment (AronNovak) [reply robo-components/DeploymentTrait.php:566]: Added.

**Pull request merged**